### PR TITLE
Fix a bug in make_function_space.jl

### PR DESCRIPTION
**Pull request by szy21**

### CODE REVIEW

**Comment by szy21** [comment]
bors try

File: src/Domains/make_function_space.jl [hunk 1]
```diff
@@ -30,7 +30,7 @@ function make_function_space(domain::Plane{FT}) where {FT}
         domain.nelements[2],
     )
     grid_topology = Topologies.GridTopology(mesh)
-    quad = Spaces.Quadratures.GLL{domain.npolynomial}()
+    quad = Spaces.Quadratures.GLL{domain.npolynomial + 1}()
```
Comment (bischtob) [inline src/Domains/make_function_space.jl:33]: This will break some test. You could just adjust the tests by lowering the poly order by 1 in the tests.

Comment (szy21) [reply src/Domains/make_function_space.jl:33]: Done!

**Comment by szy21** [comment]
bors try

**Pull request merged**